odoo/_monkeypatches/__init__.py

```diff
@@ -0,0 +1,41 @@
+# ruff: noqa: F401, PLC0415
+# ignore import not at top of the file
+import os
+import time
+from .evented import patch_evented
+
+
+def set_timezone_utc():
+    os.environ['TZ'] = 'UTC'  # Set the timezone
+    if hasattr(time, 'tzset'):
+        time.tzset()
+
+
+def patch_werkzeug():
+    from .monkeypatches_werkzeug import patch_werkzeug
+    from werkzeug import urls
+    patch_werkzeug()
+    if not hasattr(urls, 'url_join'):
+        from .monkeypatches_urls import patch_urls
+        patch_urls()
```

Comment:
let's group monkeypatches_urls and monkeypatches_werkzeug together in a same file.

```py
def patch_werkzeug():
    FileStorage.save = lambda self, dst, buffer_size=(1 << 20): copyfileobj(self.stream, dst, buffer_size)

    Request.json_module = Response.json_module = scriptsafe

    if Rule_get_func_code:
        @staticmethod
        def _get_func_code(code, name):
            assert isinstance(code, CodeType)
            return Rule_get_func_code(code, name)
        Rule._get_func_code = _get_func_code

    urls.url_decode = url_decode
    urls.url_encode = url_encode
    urls.url_join = url_join
    urls.url_parse = url_parse
    urls.url_quote = url_quote
    urls.url_unquote = url_unquote
    urls.url_quote_plus = url_quote_plus
    urls.url_unquote_plus = url_unquote_plus
    urls.url_unparse = url_unparse
    urls.URL = URL
    from ..tools.json import scriptsafe  # noqa: PLC0415
```